src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSender.java

```diff
@@ -49,21 +50,27 @@
             .register();
 
     protected final String th2Pin;
+    private final AtomicReference<ConnectionManager> connectionManager = new AtomicReference<>();
     private final AtomicReference<String> routingKey = new AtomicReference<>();
     private final AtomicReference<String> exchangeName = new AtomicReference<>();
-    private final AtomicReference<ConnectionManager> connectionManager = new AtomicReference<>();
+    private final String queueName;
+    private final long sizeLimit;
+    private final int recheckSizeTimeoutSeconds;
```

Comment:
Let's use an exponential function to calculate the size of the dynamic pause. Configuration can contain the upper limit of pause

Reply:
Sorry, could you please explain it in detail?

Reply:
Please use Math.exp(<iteration number>) + <random value from 10 to 100> to calculate the next sleep interval in milliseconds. Switch to the max value from config after the calculated value exceeded the max value

Reply:
done